test/config/domains.test.js

```diff
@@ -0,0 +1,35 @@
+import domains from '../../src/config/domains';
+
+describe(`domains config file`, () => {
+  const tDomains = {
+    departments: ['list', 'info'],
+    users: ['me', 'list', 'info'],
+    customFieldDefinitions: ['list', 'info'],
+    workTypes: ['list'],
+    contacts: ['list', 'info', 'add', 'update', 'delete', 'tag', 'untag', 'linkToCompany', 'unlinkFromCompany'],
+    companies: ['list', 'info', 'add', 'update', 'delete', 'tag', 'untag'],
+    businessTypes: ['list'],
+    tags: ['list'],
+    deals: ['list', 'info', 'create', 'update', 'move', 'win', 'lose', 'delete'],
+    lostReasons: ['list'],
+    dealPhases: ['list'],
+    dealSources: ['list'],
+    quotations: ['info', 'download'],
+    events: ['list', 'info', 'create', 'update', 'cancel'],
+    activityTypes: ['list'],
+    invoices: ['list', 'info', 'download', 'draft', 'update', 'copy', 'book', 'delete', 'registerPayment'],
+    creditNotes: ['list', 'info', 'download'],
+    taxRates: ['list'],
+    paymentTerms: ['list'],
+    withholdingTaxRates: ['list'],
+    productCategories: ['list'],
+    projects: ['list', 'info', 'create', 'update', 'delete', 'addParticipant', 'updateParticipant'],
+    milestones: ['list', 'info', 'create', 'update'],
+    timeTracking: ['list', 'info', 'add', 'update', 'resume', 'delete'],
+    timers: ['current', 'start', 'stop', 'update'],
+  };
+
+  it(`should equal the provided list of domains`, () => {
+    expect(domains).toEqual(tDomains);
```

Comment:
Isn't this a very useless test? 🤔

Reply:
It's an extra check to make sure you didn't accidentally break something while adding a new domain to `domains.js` I guess?

I'm not 100% sure myself. I can remove it, no problem.

Reply:
I would remove it. If you accidentally break something in it we'll notice it in the PR.
Now if you want to add something, you just have to add it in 2 places, which feels a bit weird.

Reply:
There is a top level test check in main too, just to be sure all domains are exposed on the API object. That one makes sense IMO @lowiebenoot?

Reply:
That makes more sense indeed.
This test was useless to me because it just checked if 2 objects were the same, without doing any operations on it.